region/crateria/east.json

```diff
@@ -2452,6 +2455,29 @@
                 }
               ]
             },
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "Space Jump Water Escape with Ice",
+                  "notable": false,
+                  "requires": [
+                    "canSpaceJumpWaterEscape",
+                    "canTrickyUseFrozenEnemies"
+                  ],
+                  "note": "Freeze the Choot and stand on it on the right side of the stalagmite. Jump to the water line and space jump to the Kamer platform."
```

Comment:
This has to be done in a very particular way.
Its a tricky jump to the right to get above the choot.
Break momentum to align with the wall.
And a small frame window shot timing on the freeze.

But attempts are repeatable without getting hit.
Could be made a notable.